Tools/TestWebKitAPI/Tests/WebKitCocoa/FontAttributes.mm

```diff
@@ -30,18 +30,41 @@
 #import "NSFontPanelTesting.h"
 #import "PlatformUtilities.h"
 #import "Test.h"
+#import "TestNavigationDelegate.h"
 #import "TestWKWebView.h"
 #import <WebCore/ColorCocoa.h>
 #import <WebCore/FontCocoa.h>
 #import <WebKit/WKUIDelegatePrivate.h>
+#import <WebKit/WKWebViewPrivate.h>
 #import <cmath>
+#import <pal/spi/cf/CoreTextSPI.h>
 #import <pal/spi/cocoa/NSAttributedStringSPI.h>
 #import <wtf/Vector.h>
 
 #if PLATFORM(IOS_FAMILY)
 #import "UIKitSPI.h"
 #endif
 
+bool _willSetFont = false;
+bool _doneAppleFontTextStyleTest = false;
+
+@interface AppleFontTextStyleUIDelegate : NSObject <WKUIDelegatePrivate>
```

Comment:
NIT: Is this really "Apple's font text style"?  Or is there a more non-specific name for this (e.g. "Title font text style")?

Reply:
@dcrousso Ya I thought it was a bad name too, but couldn't think of one. It's not just for titles, its for any of the system font styles such as callout, body, etc

Reply:
I named it `AppleFontTextStyle...` because the actual names of the styles are like `-apple-system-title2` etc